src/incubator/Dialog/useDialogContent.tsx

```diff
@@ -49,7 +49,7 @@ const useDialogContent = (props: InternalDialogCloseButtonProps) => {
     return showCloseButton ? [propsContainerStyle, styles.transparent] : propsContainerStyle;
   }, [showCloseButton, propsContainerStyle]);
 
-  const DialogContent = () => {
+  const getDialogContent = () => {
```

Comment:
let's rename it to`renderDialogContent` because it's a render method (it returns an element)